### ISSUE TITLE
Export step fails to open file

### ISSUE THREAD

**Issue by jacobalbano** [audiamus/BookLibConnect#37]
Hi there, getting the following error message when I try to export a file.

![image](https://user-images.githubusercontent.com/3813170/161440529-31559ee3-80ad-4a22-aaa3-c74297d0e9c8.png)

The download and decryption steps work correctly and I can see the files in my download folder:

```
C:\temp\Audible\Download
λ dir
 Volume in drive C has no label.
 Volume Serial Number is 34EA-5379

 Directory of C:\temp\Audible\Download

04/03/2022  12:30 PM    <DIR>          .
04/03/2022  12:30 PM    <DIR>          ..
04/03/2022  12:30 PM       431,185,907 [3巻] 青春ブタ野郎はロジカルウィッチ_B0854L1FC2_LC_128_44100_Stereo.aaxc
04/03/2022  12:30 PM       425,106,042 [3巻] 青春ブタ野郎はロジカルウィッチ_B0854L1FC2_LC_128_44100_Stereo.m4b
               2 File(s)    856,291,949 bytes
               2 Dir(s)  286,003,892,224 bytes free

C:\temp\Audible\Download
```

(The reason I'm using C:\temp is because if I use my actual Music folder, which lives on a different drive, the process silently fails).

Attached are some relevant files. Please let me know if there's any more information I can provide.

[BookLibConnect_2022-04-03_005.log](https://github.com/audiamus/BookLibConnect/files/8405163/BookLibConnect_2022-04-03_005.log)
[usersettings.json](https://github.com/audiamus/BookLibConnect/files/8405164/usersettings.json.txt)
[stacktrace.txt](https://github.com/audiamus/BookLibConnect/files/8405165/stacktrace.txt)

**Comment by audiamus** [comment]
I thought I had fixed this. Probably the same cause as #34. I am using a low level Win32 copy mechanism for this which has its own ideas about character encoding. I have tried with Cyrillic and that seemed to work. Will test with Japanese again.

Regarding the external drive: Was that a local external drive, USB stick or memory card, or a drive on a remote system, NAS, cloud etc? Was it mounted by drive letter?

**Comment by jacobalbano** [comment]
> Regarding the external drive: Was that a local external drive, USB stick or memory card, or a drive on a remote system, NAS, cloud etc? Was it mounted by drive letter?

It's actually an internal drive; I have it mounted as F:\ and keep my music, documents etc on it. For example my Music library is located at F:\Music, etc.
![image](https://user-images.githubusercontent.com/3813170/161447416-5be8a8ea-0c34-4a6e-90b7-4d904402a707.png)

**Comment by audiamus** [comment]
An internal drive should be completely harmless. And your Windows system language is English? It shouldn't matter with Unicode  but you never know.

**Comment by jacobalbano** [comment]
Correct, my system language is English.

**Comment by audiamus** [comment]
Well, it appears I had indeed already fixed it, but then cleaned up the code and by that, the Unicode enforcement was lost again, unfortunately.  Hence my Cyrillic test now failed as well. It will be fixed with the next update.

As a temporary workaround, you can manually copy the .m4b file to the export directory and rename it to .aax. 

Nonetheless, I am still curious about what might have happened when you were using your F: drive. I will add explicit exception handling for this bit.

**Comment by jacobalbano** [comment]
Thanks! If you add the exception handling in the next update I can try my old configuration again and open a new issue if I find anything.

**Comment by audiamus** [comment]
Fix in[ version 0.8](https://github.com/audiamus/BookLibConnect/releases/tag/v0.8).

**Comment by jacobalbano** [comment]
Working perfectly now, thanks so much!
